common/reviews/api/rush-lib.api.ts

```diff
@@ -58,13 +58,12 @@ enum BumpType {
 
 // @public
 class ChangeFile {
```

Comment:
Define this API

```
class ChangeLog {
  public static createEmptyChangeFile(projectName: string): string {
  }
}
```


AND remove all the other change-related classes that aren't realistically usable.